src/main/java/ch/njol/skript/effects/EffToggleFlight.java

```diff
@@ -50,7 +50,7 @@
 
 	private boolean allow;
 
-	@SuppressWarnings("unchecked")
+	@SuppressWarnings({"unchecked", "null"})
```

Comment:
why is this here

Reply:
It creates a warning, use the proper IDE to view.

Reply:
There is no "proper" IDE, even IntelliJ can warn about these things if configured that way, so there is no need to debate about something that's completely personal such as IDE preference.

Reply:
Then it should have been seen as these were new additions.